ray_on_golem/server/services/golem/helpers/demand_config.py

```diff
@@ -32,28 +30,14 @@ async def get_payload_from_demand_config(self, demand_config: DemandConfigData)
     async def _get_repository_payload(
         self, demand_config: DemandConfigData, image_url: URL, image_hash: str
     ) -> Payload:
-        @dataclass
-        class CustomRepositoryVmPayload(RepositoryVmPayload):
-            subnet_constraint: str = constraint("golem.node.debug.subnet", "=", default=SUBNET)
-            debit_notes_accept_timeout: int = prop(
-                "golem.com.payment.debit-notes.accept-timeout?", default=240
-            )
-
         params = demand_config.dict(exclude={"image_hash", "image_tag", "outbound_urls"})
```

Comment:
Should we exclude`subnet_tag` here too?

Reply:
hmm, indeed... though, it seems to work this way too :D

Reply:
maybe...

Reply:
but it works all the same anyway :D

Reply:
So why add it to the one below?

Reply:
that's a good question :)

Reply:
anyway, updated